Client/src/Application/Application.cpp

```diff
@@ -8,21 +8,31 @@
 #include "Application.hpp"
 #include "ClientSceneManager.hpp"
 
+void Application::_packetHandler(Network::UDPPacket packet)
+{
+    std::cout << "Messages received from server: [...]" << std::endl;
+}
+
+
 Application::Application()
 {
     _registries = std::make_shared<std::vector<ECS::Registry>>();
     ECS::Registry reg;
     _registries.get()->push_back(reg);
     SceneManager::ClientSceneManager sceneManager(_registries);
-    // Initialize the network and the first scene
+
+    _client = std::make_shared<Network::Client>("127.0.0.1", 4444, 4445);
+    _client->connect([this](Network::UDPPacket packet) {
+        this->_packetHandler(std::move(packet));
+    });
```

Comment:
The client don't choose where he can be connected ? R-type can be an online game

Reply:
Now we can't choose  where he can be connected so we have to wait the graphic so that the user can entered where he wanrt to be connected